ExportScripts/export_mps.py

```diff
@@ -0,0 +1,248 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of https://www.github.com/o3de/o3de.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import argparse
+import logging
+import sys
+
+import o3de.export_project as exp
+import o3de.enable_gem as enable_gem
+import o3de.disable_gem as disable_gem
+import o3de.manifest as manifest
+import pathlib
+
+
+def export_multiplayer_sample(ctx: exp.O3DEScriptExportContext,
```

Comment:
is this function actually different from the one in the project export script in o3de engine repo? If not, you could probably just use that, and inject the necessary arguments as hardcoded values (which should be fine for MPS, since it already needs to define what is required)

Reply:
I would, but this one adds the injection of the MPSGamelift gem as an option (check with @gene-walt ), so thats why the original couldn't be used

Reply:
ah okay, in that case we can probably keep it like this for now